public/app/plugins/datasource/cloudwatch/components/VariableQueryEditor/MultiFilterItem.tsx

```diff
@@ -0,0 +1,63 @@
+import { css, cx } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+import { AccessoryButton, InputGroup } from '@grafana/experimental';
+import { Input, stylesFactory, useTheme2 } from '@grafana/ui';
+import React, { FunctionComponent, useState } from 'react';
+import { MultiFilterCondition } from './MultiFilter';
+
+export interface Props {
+  filter: MultiFilterCondition;
+  onChange: (value: MultiFilterCondition) => void;
+  onDelete: () => void;
+}
+
+export const MultiFilterItem: FunctionComponent<Props> = ({ filter, onChange, onDelete }) => {
+  const [localKey, setLocalKey] = useState(filter.key || '');
+  const [localValue, setLocalValue] = useState(filter.value?.join(', ') || '');
+  const theme = useTheme2();
+  const styles = getOperatorStyles(theme);
+
+  return (
+    <div data-testid="cloudwatch-multifilter-item">
+      <InputGroup>
+        <Input
+          data-testid="cloudwatch-multifilter-item-key"
+          aria-label="Filter key"
+          value={localKey}
+          placeholder="key"
```

Comment:
can you make the placeholder an optional property? that way the parent component can use more meaningful examples (like an example attribute for the ec2 instance or a tag for the arn resource)